packages/web/components/Guild/GuildTile.tsx

```diff
@@ -36,7 +37,7 @@ export const GuildTile: React.FC<Props> = ({ guild }) => (
           <Heading
             size="lg"
             color="white"
-            bgColor="whiteAlpha.100"
+            // bgColor="landingGlassDark" // This colour doesn't exist in the design system, should be landingDarkGlass
```

Comment:
If it should be `landingDarkGlass`, why is it commented out?

```suggestion
            bgColor="landingDarkGlass"
```